src/client/opamLockCommand.ml

```diff
@@ -252,7 +254,7 @@ let lock_opam ?(only_direct=false) st opam =
   let all_depopts =
     OpamFormula.packages st.packages
       (OpamFilter.filter_deps
-         ~build:true ~test:true ~doc:true ~dev:true ~default:true ~post:false
+         ~build:true ~test:true ~doc:true ~wdev:true ~default:true ~post:false
```

Comment:
why did `opam lock` had `dev` to `true` in the first place? This feels a bit weird

Reply:
i may say to have all dependencies, and filter afterwards?